### ISSUE TITLE
Training stuck

### ISSUE THREAD

**Issue by Gabrielk99** [ditschuk/pytorch-tsmixer#4]
Hi, first of all, thanks for you implementation of TSMixerExtended

I have some doubt, I initialize the model and start the training process, but the predictions start much lower, and grow up slow each epoch, the training process get stuck on some local minimun aparently.

Exist some torch process that I have to do before? I am new using torch

**Comment by LVystorobska** [comment]
hi! it can be because of the learning rate, especially if you used optimizer initialization from readme example; for me, simply increasing the learning rate (lr param) worked. However training process seems to be a bit unstable still, i did not use any data scaling beforehand though, that can also can be the issue in the described case. Did you manage to resolve it anyway, @Gabrielk99 ?
